Adding v1 of the EULA

joachimweyl opened the pull request:
Please confirm v1.0.1 is the correct version
prior to approving this PR

joachimweyl committed:
Adding v1 of the EULA

Please confirm v1.0.1 is the correct version
prior to approving this PR

joachimweyl commented on EULA.md:
```diff
@@ -0,0 +1,288 @@
+Services. Any material modifications and additional terms and conditions
+will be communicated to you and, if accepted, will be effective
+immediately and will be incorporated into this Agreement. In the event
+that you refuse to accept such changes, the Provider will have the right
```
This wording might need to be stronger. Do we need to be clear that if they don't accept the changes they must terminate the agreement? The current wording sounds like we have to be the ones to first find out if they accepted and second terminate if they did not.
Something along the lines of "... will be communicated to you and, will be effective immediately and incorporated into this Agreement. In the event that you refuse to accept such changes, you are required to terminate this Agreement pursuant to Section 12.

msdisme replied:
Follow up discussion with Kim: we want to, if it is ok to, put onus on them to cancel since they  may not be forced through log in which would require them to "sign off" for long periods of time as they may interface via programs only.

joachimweyl replied:
addressed by a change to lines 15-19, now 15-22

msdisme commented on EULA.md:
```diff
@@ -0,0 +1,288 @@
+Alliance (the “Services”). Each Service is provided and managed by the
+MOC Alliance Collaborator(s) identified [here](#service_name) (each a
+“Provider”).
+
```
Suggestion from OGC to change line 15-19: 
This End User License Agreement (the “Agreement”) applies to all uses of software and services rendered by and in relation to the Mass Open Cloud Alliance (the “Services”). Each Service is provided and managed by the MOC Alliance Collaborator(s) identified here (each a “Provider”). **The Providers may change this Agreement at any time, and we will try to let you know as soon as possible about any new changes. If you do not wish to accept the new Agreement, you must immediately stop using the Services and terminate this Agreement pursuant to Section 12.**

msdisme replied:
to adress items below

joachimweyl committed:
Update based on OGC suggestion

msdisme reviewed (approved):
Looks good - Lilly gave us a way to address concerns raised about notification.

larsks commented on EULA.md:
```diff
@@ -0,0 +1,291 @@
+Alliance Collaborators, please visit*
+*[here](https://www.bu.edu/hic/centers-initiatives-labs/mocalliance/)*.
+
+**MASS OPEN CLOUD ALLIANCE (MOC ALLIANCE) END USER LICENSE AGREEMENT**
```
This (and all other section headings) should be actual section heads:

```
# MASS OPEN CLOUD ALLIANCE (MOC ALLIANCE) END USER LICENSE AGREEMENT
```

larsks commented on EULA.md:
```diff
@@ -0,0 +1,291 @@
+otherwise authorized in writing by the Provider for a specified research
+purpose):
+
+1) probe, scan, or test the vulnerability of any system or network;
```
This numbered list should be an actual Markdown numbered list:

```
1. probe, scan, or test the vulnerability of any system or network;
2. breach or otherwise circumvent any security or authentication measures;
.
.
.
```

larsks commented on EULA.md:
```diff
@@ -0,0 +1,291 @@
+    Regulation, China’s Personal Information Protection Law, the U.S.’s
+    HIPAA).
+
+19. **Removal of Content**
```
This looks like an error (it should be a section heading, not the 19th item of the preceding list).

larsks commented on EULA.md:
```diff
@@ -0,0 +1,291 @@
+courts located in Boston, Massachusetts, and you hereby submit to, and
+waive objection to, the jurisdiction and venue of such courts.
+
+**<u>Information to be provided at link </u>**:
```
Drop the `<u>...</u>`...and of course this should probably just be a section heading
.

larsks commented on EULA.md:
```diff
@@ -0,0 +1,291 @@
+collaborative effort between higher education, non-profit entities and
+industry (MOC Alliance Collaborators). For a full list of the MOC
+Alliance Collaborators, please visit*
+*[here](https://www.bu.edu/hic/centers-initiatives-labs/mocalliance/)*.
```
Avoid links on the word "here" for documents that might be printed. Show the actual URL or section name instead:

```
...please visit <https://www.bu.edu/hic/centers-initiatives-labs/mocalliance/>.
```

larsks committed:
Correct markdown formatting issues

This makes several formatting changes to the EULA:

- Section heads are actually section headings
- Numbered lists are actually numbered lists
- I have reformatted all links to be useful when printed
- I have reformatted the contact information table as a series of lists
  in order to make it more manageable

larsks commented:
Apparently I **do** have the permissions to update this PR, so I'm closing #3.

larsks commented on EULA.md:
```diff
@@ -0,0 +1,335 @@
+courts located in Boston, Massachusetts, and you hereby submit to, and
+waive objection to, the jurisdiction and venue of such courts.
+
+## Contact Information
```
This section was previously titled, "Information to be added in the link", which didn't make any sense. I've renamed it to "Contact Information", but I'm not wedded to that.

knikolla commented on EULA.md:
```diff
@@ -0,0 +1,335 @@
+
+This Agreement may be terminated by the Provider for any reason upon 48 hours’
+notice to you. In such an event, you will be afforded a reasonable opportunity
+to retrieve your User Content from the Services. If your account access has
```
Need to be mindful not to initiate operations that may cause deletion of user resources (like setting an OpenShift allocation status in ColdFront to "Denied".

msdisme replied:
Will need to be adressed via process and doc, I will mention it to Milson.

knikolla reviewed (commented):
Looks good to me after reading it, but I'm not a lawyer and I'm unsure who ultimately should click the approve button.

msdisme reviewed (approved):
Approving in current state so we may move it forward however a number of changes will still need to be made here.  1. removal of 'here' links breaks the meaning of contract so need to modify appropriately 2. contract refers to contract sections by number so need to explore if they must be specific of if a semantic way to include would work.

larsks committed:
Correcting intra-document references

Change numbered section references to links and correct some other instances
of "here" that used to be links in the original document.
